tests/scripts/quiet/quiet

```diff
@@ -1,5 +1,3 @@
-#! /usr/bin/env bash
-#
 # Copyright The Mbed TLS Contributors
```

Comment:
It would be nice to have some indication that this script uses bash. For example you can declare it to Emacs by putting this on the first line:
```
# -*-mode: sh; sh-shell: bash -*-
```
(Would be nice to have the Vim equivalent but I don't know what it is.)